src/cluster/src/delete.rs

```diff
@@ -243,18 +249,36 @@ impl ClusterUninstaller {
             info!("deleting all {} in: {}", object_type, namespace);
             cmd.arg("--all");
```

Comment:
So I suggest check if if succeed or not, if fail, don't return error.   In fact, signature could be as:
```
fn remove_custom_objects(
         &self,
         object_type: &str,
         namespace: &str,
         selector: Option<&str>,
     ) -> Result<bool, UninstallError>
```

If it failed to delete, return false

Reply:
I actually just made everything ignore the results of deleting, which I think does what we need